contracts/V1/StateChannelDiamondProxy/StateChannelManagerProxy.sol

```diff
@@ -384,6 +385,41 @@ contract StateChannelManagerProxy is StateChannelManagerInterface, StateChannelC
         return disputeWindow.evidence.disputeCommitments;
     }
 
+    function getDisputeWindowCreationTimestamp(bytes32 channelId, bytes32 forkId)
+        public
+        view
+        returns (uint256 creationTimestamp)
+    {
+        DisputeData storage _disputeData = disputeData[channelId];
+        DisputeWindow storage disputeWindow = _disputeData.disputeWindowMap[forkId];
+        return disputeWindow.evidence.creationTimestamp;
+    }
+
+    function reduce(Dispute[] memory disputes, uint256 disputeWindowCreationTimestamp)
+        public
+        view
+        returns (ReduceOutput memory reducedOutput)
+    {
+        return disputeVerificationFacet.reduce(disputes, disputeWindowCreationTimestamp);
```

Comment:
this needs to be a `delegatecall` so it executes in the storage context of the `StateChannelManagerProxy` like all other operations